docs/source/references/modules.rst

```diff
@@ -52,12 +52,12 @@ SumMetric
 .. autoclass:: torchmetrics.SumMetric
     :noindex:
 
-*************
-Audio Metrics
-*************
+*****
+Audio
+*****
 
-About Audio Metrics
-~~~~~~~~~~~~~~~~~~~
+About Audio
+~~~~~~~~~~~
```

Comment:
I think this is not needed as it not with other section neither

Reply:
Removed it in my latest commit.